BookPlayer/Player/PlayerViewController.swift

```diff
@@ -177,7 +177,11 @@ class PlayerViewController: UIViewController, UIGestureRecognizerDelegate {
             items.append(self.sleepLabel)
         }
 
-        self.chaptersButton.isEnabled = self.currentBook.hasChapters
+        guard let chapterCount = self.currentBook.chapters?.count else {
```

Comment:
This `guard` here if the `count` ever resolves to `nil` then the button `avRoutePickerBarButtonItem` after this block would never be added. Testing it, I found out that for books w/o chapters, it resolves to `0`, but I think that it's better if we change this guard and make it more explicit by replacing it for:
```swift
let chapterCount = self.currentBook.chapters?.count ?? 0
```